CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(freud)
+
+set(DEFAULT_BUILD_TYPE "Release")
+
+# Set a default build type if none was specified
+if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+    message(STATUS "Setting build type to '${DEFAULT_BUILD_TYPE}' since none was specified.")
+    set(CMAKE_BUILD_TYPE "${DEFAULT_BUILD_TYPE}" CACHE
+        STRING "Choose the type of build." FORCE)
+    set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
+endif()
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Got it, I had issues with it early on but it might have been because I was misusing OBJECT and SHARED in my early iterations. It does indeed seem to work correctly on shared libraries without this.